### DISCUSSION TITLE
Welche Anlagen ID wird erwartet beim Start von Solectrus?

### DISCUSSION THREAD

**Discussion by dklauer** [solectrus/solectrus#3298] [General]
Hallo und herzlichen Dank für die super App!

Bei uns wurde in den vergangenen Tagen der Home 4 ausgetauscht. Damit haben wir eine neue Gehäuse und Steuereinheitsnummer bekommen und dementsprechend 2 Anlagen in mein-senec.de bzw. in der SENEC App sichtbar. 

Meine Frage, welche ID erwartet Solectrus um zwischen den Anlagen zu unterscheiden und die richtigen Werte bei mein-senec.de abzurufen?
Die Gehäuseseriennummer (S4H1-...)? --> läuft in "Error getting data from SENEC cloud: Error 400"
Die Steuereinheitennummer (v4-000...)? --> läuft in "Error getting data from SENEC cloud: Error 400"
Die Steuereinheitennummer ohne v4- (000...)?  --> läuft in "Error getting data from SENEC cloud: Error 401"
Die Anlagennummer, die nur über die Developertools des Browsers ersichtlich sind, wenn eine der Anlagen über mein-senec.de ausgewählt werden? In meinem Fall "0" oder "1".  --> läuft in "Error getting data from SENEC cloud: Error 400"

Vielen Dank vorab!

**Comment by ledermann** [comment]
Es wird die SYSTEM-ID erwartet, die meines Wissens nach nirgendwo (?) von SENEC angezeigt wird.

Beim **Start** des SENEC-Collector wird sie aber ausgegeben, beispielsweise sieht das so aus:

```
SENEC collector for SOLECTRUS, Version develop, built at 2024-07-25T23:52:23.310Z
https://github.com/solectrus/senec-collector
Copyright (c) 2020-2024 Georg Ledermann, released under the MIT License
Using Ruby 3.3.4 on platform x86_64-linux-musl
Pushing to InfluxDB at http://influxdb:8086, bucket SENEC, measurement SENEC
Pulling from SENEC cloud every 300 seconds
No SENEC_SYSTEM_ID given, looking for existing systems...
Found SENEC system 123456 (S439123456789012345, DE-V3-HD-03LI10-01234)
Using first system, which is 123456
```

Die System-ID lautet in diesem (Fake-)Beispiel also `123456`. Gibt man nicht explizit eine ID an, wird die erste gefundene genommen. Wenn du zwei IDs hast, musst du die zu benutzende Nummer angeben. Es wäre ein solcher Eintrag in der `.env` notwendig:

```env
SENEC_SYSTEM_ID=123456
```

Nicht vergessen, diese Variablen in der `docker-compose.yml` mit aufzuführen!

**Reply by dklauer** [reply]
Danke für den Hinweis! 

Einmal ohne Werte für die SENEC_SYSTEM_ID starten brachte die Erleuchtung. :)
